controllers/module_reconciler.go

```diff
@@ -357,6 +353,32 @@ func (r *ModuleReconciler) handleDevicePlugin(ctx context.Context, mod *kmmv1bet
 	return err
 }
 
+func (r *ModuleReconciler) garbageCollect(ctx context.Context,
```

Comment:
Do we want to collect signing jobs as well?

Reply:
yes, but it will be implemented after all Chris' PRs are completed